tools/clang/test/CodeGenSPIRV/hull.pcf.void.hlsl

```diff
@@ -0,0 +1,32 @@
+// Run: %dxc -T hs_6_0 -E main
+
+#include "bezier_common_hull.hlsl"
+
+// CHECK: [[fType:%\d+]] = OpTypeFunction %HS_CONSTANT_DATA_OUTPUT
+// CHECK:       {{%\d+}} = OpFunctionCall %HS_CONSTANT_DATA_OUTPUT %PCF
```

Comment:
I'd suggest to add `// CHECK-LABEL: %main = OpFunction` or some comments to make it clear the following is in the wrapper.